### ISSUE TITLE
Linda: Chat Server

### ISSUE THREAD

**Issue by lindalonglong** [evagravin/p4-kangaroos#36]

**Comment by avabrooks** [comment]
I like how you created two different tickets for the frontend and backend of the chat server, it's a large part of the project so it's smart to split it up. Maybe under each sub-bullet point, you can be a bit more specific with what you're going to do code-wise. This is a large part of the project so it would be beneficial to maybe write out smaller steps as well.

**Comment by risaiwazaki** [comment]
I agree with Ava that sub-bullets under each topic would be helpful in knowing what you still have left to do. Smacking other accounts as well as messaging is s key factor in this social media account so it's smart that you thought of the process and how to connect it with what we currently have!
